### ISSUE TITLE
Regression by v1.0.4: Commented-out heredocs

### ISSUE THREAD

**Issue by indocomsoft** [rrrene/credo#646]
### Environment

* Credo version (`mix credo -v`): 1.0.4
* Erlang/Elixir version (`elixir -v`):
```
Erlang/OTP 21 [erts-10.3.1] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.8.1 (compiled with Erlang/OTP 21)
```

* Operating system: macOS 10.14.4 Mojave

### What were you trying to do?
Running `mix credo`.

### Expected outcome
Before 1.0.4, Credo does not report any error on files with commented out heredocs, https://github.com/source-academy/cadet/blob/master/lib/cadet/course/course.ex#L98.

### Actual outcome
Since 1.0.4, Credo complains about trailing white-spaces:
```
  Code Readability
┃
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:205:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:189:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:188:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:175:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:157:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:150:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:138:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:125:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:124:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:100:1 #()
┃ [R] ↘ There should be no trailing white-space at the end of a line.
┃       lib/cadet/course/course.ex:99:1 #()
```

If I delete the commented out code in the file, credo 1.0.4 does not complain about anything. However, Credo 1.0.3 and before worked just fine even with the commented out code in the file.

I bisected the regression to commit 7a92ac27706dd509abf9cf31d0a89510b7ea704c

**Comment by rrrene** [comment]
> I bisected the regression to commit [7a92ac2](https://github.com/rrrene/credo/commit/7a92ac27706dd509abf9cf31d0a89510b7ea704c)

Ha, great detective work! :+1:

**Comment by rrrene** [comment]
@indocomsoft  Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by indocomsoft** [comment]
That seems to solve the issue. Thank you very much!

Will be looking forward to this being released on hex

**Comment by jiachen247** [comment]
@rrrene thanks for the hard work! when will this be released on hex?

**Comment by rrrene** [comment]
@indocomsoft @jiachen247 It's on hex now: https://hex.pm/packages/credo/1.0.5

Sorry for the delay! :+1:
